### PR TITLE
Only report designer attributes for the primary project of a multi-project config.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Roslyn part of https://devdiv.visualstudio.com/DevDiv/_queries/edit/961596/?triage=true

Project system still needs to do work once this is in.

### CODE REVIEW

File: src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs [hunk 1]
```diff
@@ -68,6 +69,13 @@ internal sealed class VisualStudioProject
         private string? _outputRefFilePath;
         private string? _defaultNamespace;
 
+        /// <summary>
+        /// We store if this is a primary project or not as a tri-state.  Only <see langword="false"/> means that it's
+        /// not a primary project.  This way, if the value hasn't been set, or is never set, we still consider the
+        /// project to be a primary one.
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs:75]: How is this different than just starting it out assigned as 'true' and removing the tri-state?

File: src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs [hunk 2]
```diff
@@ -289,6 +297,12 @@ internal bool HasAllInformation
             set => ChangeProjectProperty(ref _hasAllInformation, value, s => s.WithHasAllInformation(Id, value));
         }
 
+        internal bool IsPrimary
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs:300]: Any tests for this? There's already a harness set up in https://github.com/dotnet/roslyn/tree/master/src/VisualStudio/Core/Test/ProjectSystemShim/VisualStudioProjectTests so just add another file and maybe add something small?

File: src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs (hunk 1 above)
Comment (CyrusNajmabadi) [reply src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs:75]: that makes sense :)

File: src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs (hunk 2 above)
Comment (CyrusNajmabadi) [reply src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs]: sure.

**Comment by davkean** [comment]
Note, our counter change is here: https://github.com/dotnet/project-system/pull/6567.

**Pull request merged**